src/playground/service.clj

```diff
@@ -71,7 +75,9 @@
     ["/api" :get (into component-interceptors [http/json-body (param-spec-interceptor ::api :query-params) `api])]
     ["/invoices/insert" :get (into component-interceptors [http/json-body (param-spec-interceptor ::invoices.insert/api :query-params) `invoices.insert/perform])]
     ["/invoices/:id" :get (into component-interceptors [http/json-body (param-spec-interceptor ::invoices.retrieve/api :path-params) `invoices.retrieve/perform])]
```

Comment:
/invoices/:id is the route I'd like to test. I created  `CREATE TABLE users (id INTEGER, email VARCHAR(50));` in the DB.